Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/InputValue.cs

```diff
@@ -35,6 +35,43 @@ public object Get()
         }
 
         ////TODO: add automatic conversions
+        /// <summary>
+        /// Read the current value of the action.
+        /// </summary>
+        /// <returns>The current value from the action cast to the specified type.</returns>
+        /// <typeparam name="TValue">Type of value to read. This must correspond to the
+        /// expected by either <see cref="control"/> or, if it is a composite, by the
+        /// <see cref="InputBindingComposite"/> in use.
+        /// The type depends on what type of controls the action is bound to. Common types are <c>float</c> and <see cref="UnityEngine.Vector2"/></typeparam>
+        /// <exception cref="InvalidOperationException">The given type <typeparamref name="TValue"/>
+        /// does not match the value type expected by the control or binding composite.</exception>
+        /// <remarks>
+        /// The following example shows how to read a value from a <see cref="PlayerInput"/> message.
+        ///
+        /// <example>
+        /// <code>
+        /// [RequireComponent(typeof(PlayerInput))]
+        /// public class MyPlayerLogic : MonoBehaviour
+        /// {
+        ///     private Vector2 m_Move;
+        ///
+        ///     // 'Move' input action has been triggered.
+        ///     public void OnMove(InputValue value)
+        ///     {
+        ///         // Read value from control. The type depends on what type of controls the action is bound to.
+        ///         m_Move = value.Get&lt;Vector2&gt;();
```

Comment:
same here (see comment below)

Reply:
I will correct the name from OnUpdate to Update in this example.